en/python_installation/instructions.md

```diff
@@ -17,6 +17,10 @@ In upcoming steps, you'll be using the Windows Command Line (which we'll also te
 
 ![Type "cmd" in the "Run" window](../python_installation/images/windows-plus-r.png)
 
+Note: if you are using an older version of Windows (7, Vista, or any older version) and the Python 3.5.x installer fails with an error, you can try either:
+1. install all Windows Updates and try to install Python 3.5 again; or
+2. install an older version of Python, e.g., 3.4.4.
```

Comment:
Hi! It's the same as in [line 10](https://github.com/DjangoGirls/tutorial/pull/892/files#diff-192e218f57c2bf35570e309e125ed0f1R10). There was a recent change there to make it point to the general page instead of to an exact minor release. Therefore people go to https://www.python.org/downloads/windows/ (the link in line 10), then all they have to do is pick a different version.